### PR TITLE
add known issue about deleted library restore

### PR DESCRIPTION
**Pull request by ryanfkeepers**
#### Does this PR need a docs update or release note?

- [x] :white_check_mark: Yes, it's included

#### Type of change

- [x] :world_map: Documentation

#### Issue(s)

* #3382

### CODE REVIEW

File: website/docs/support/known-issues.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Below is a list of known Corso issues and limitations:
   while a backup is being created will be included in the running backup.
   Future backups run when the data isn't modified will include the data.
 
-* OneDrive files ending in `.meta` or `.dirmeta` get omitted from Details and Restore commands.
```
[the remark this answers is not in the document]
Comment (ashmrtn) [reply website/docs/support/known-issues.md:23]: As far as I know (didn't check the code or anything) those files shouldn't be shown. I was unaware not showing them was considered an issue?

[the remark this answers is not in the document]
Comment (gmatev) [reply website/docs/support/known-issues.md:23]: Right. the issue would be if they are not omitted

**Comment by ryanfkeepers** [comment]
@gmatev @ashmrtn looks like the changes are good.  Could I please get an approval?

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,6 +20,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - OneDrive backups no longer include a user's non-default drives.
 - OneDrive and SharePoint file downloads will properly redirect from 3xx responses.
 
+ ### Known Issues
+ - SharePoint document library data can't be restored after the library has been deleted.
```
Comment (ashmrtn) [inline CHANGELOG.md:24]: maybe `if the library has been deleted`? After kind of makes it sound like it will definitely happen

**Review by ashmrtn — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (vkamra) [reply CHANGELOG.md:24]: Sorry - late to review. Should add this will be addressed once support to restore to a different library than source is added.

**Pull request merged**